next.config.ts

```diff
@@ -8,10 +8,27 @@ const __dirname = dirname(__filename);
 
 const config: NextConfig = {
   /** @see {@link https://nextjs.org/docs/app/building-your-application/styling/sass} */
+  webpack(config) {
+    config.module.rules.push({
+        test: /\.svg$/,
+        use: ['@svgr/webpack'],
+    })
+    return config
+  },
   sassOptions: {
     implementation: "sass-embedded",
     includePaths: [join(__dirname, "src/styles")],
   },
```

Comment:
この上にあるドキュメントリンクの関連性がなくなっちゃうので、ここの入れ替えだけお願いします！

```suggestion
  sassOptions: {
    implementation: "sass-embedded",
    includePaths: [join(__dirname, "src/styles")],
  },
  webpack(config) {
    config.module.rules.push({
        test: /\.svg$/,
        use: ['@svgr/webpack'],
    })
    return config
  },
```